module_configuration.py

```diff
@@ -28,28 +31,74 @@ def install_client_pkg(client_path):
         print(f"pip install {client_path}")
 
 
-def configure_server_module():
-    print("Not Implemented")
+def get_server_modules(cfg):
+    mod_list = []
+    with open(cfg) as f:
+        for line in f.readlines():
+            m = line.split("#")[0].split("/")[-1].strip()
+            mod_list += [m] if m else []
+    return mod_list
+
+
+def get_chpl_files(mod_path):
+    path = mod_path + "/server/*.chpl"
+    return glob.glob(path)
+
+
+def configure_server_module(mod_path, ak_loc):
+    if not ak_loc:
+        raise RuntimeError("--ak option is required when configuring a module with server components.")
+    mod_cfg = mod_path + "/server/ServerModules.cfg"
+    ak_cfg = ak_loc + "/ServerModules.cfg"
+    if not os.path.exists(mod_cfg):
+        raise RuntimeError(f"Could not locate module ServerModules.cfg: {mod_cfg}")
+    if not os.path.exists(ak_cfg):
+        raise RuntimeError(f"Could not locate module ServerModules.cfg: {ak_cfg}")
```

Comment:
might be worth changing the second one to `"Could not locate arkouda ServerModules.cfg: {ak_cfg}"`

Reply:
Updated